appHooks/index.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * appHooks/index.js
+ * script for conducting appHooks
+ *
+ * @exports {Object} run - run appHooks with given data
+ */
+
+import Promise from 'bluebird'
+import _ from 'lodash'
+
+import { Log } from '~/app'
+
+let fs = Promise.promisifyAll(require('fs'))
+
+function getHooks (lvl) {
```

Comment:
this is a complex function, would be nice to have a little docstring header, at least explaining what the arguments are and what it returns.